storage_manager_new.py

```diff
@@ -5,46 +5,50 @@ class Storage:
 
     def __init__(self, name):
         self.name = name
+        self.items = []
+        self.load_items_from_file()
 
     def load_items_from_file(self):
-        Item.instances = []
+        self.items = []
         File = open(Storage.database_file, 'r')
         for line in File.readlines():
             line = line.rstrip('\n')
-            Item.instance_from_string(line)
+            self.items.append(Item.create_instance_from_string(line))
 
     def add_item(self):
         # collect_new_item_data
         new_items_data = []
-        add_more = True
-        while add_more == True:
-            new_item = Interface.get_user_choice(
-                'Please provide new item data in order: category, type, model, quantity (Example: Sound, Mixer, Midas M32, 1).')
+        while True:
+            new_item = input(
+                'Please provide new item data in order: category, type, model, quantity (Example: Sound, Mixer, Midas M32, 1).\n')
             if new_item.lower() != 'menu':
                 new_items_data.append(new_item)
             else:
                 break
-            ask_for_more = Interface.get_user_choice(
-                'Would you like to add another item? Please type Y/N')
+            ask_for_more = input(
+                'Would you like to add another item? Please type Y/N\n')
             if ask_for_more.lower() == 'n':
-                add_more = False
+                break
         # create_item_instance and write to database_file
         for string in new_items_data:
             try:
-                data = string.split(', ')
-                quantity = int(data.pop(3))
-                string_data = ', '.join(data)
+                list_from_string = string.split(', ')
+                quantity = int(list_from_string.pop(3))
+                string_data = ', '.join(list_from_string)
                 for i in range(1, (quantity + 1)):
                     Item.create_instance_from_string_and_write_to_database_file(
                         string_data)
+                    print(f'{string_data} was added successfully.')
             except IndexError:
                 pass
+        print('')
+        self.load_items_from_file()
 
     def show_all_items(self):
         print('')
         items_to_show = []
-        if Item.instances != []:
-            for item in Item.instances:
+        if self.items != []:
```

Comment:
As empty list is falsy in Python, I think you can just `if not self.items:` here?

Reply:
This condition has to evaluate to True while `self.items` isn't an empty list, to start showing items to the user. If `self.items` is an empty list, code from line 57 is executed. I believe, that changing condition `self.items != []:` to `if not self.items:` (while `self.items` isn't an empty list) would evaluate to False?

Reply:
Ah, yeah, but changing it to `if self.items:` would work the same, right?